### PR TITLE
Improving docker support

### PR DESCRIPTION
**Pull request by ThisIsAreku**
- Added Redis after commit eb5783697bf2b0c9fae063eec987310b47f449ae
- Forced build against specific ubuntu version
- Reduced container size
- Added .dockerignore file

### CODE REVIEW

**Comment by OrangeCrush** [comment]
:+1: 

Nice work!  Very curious to see how the Redis cache improves performance.

**Comment by phalt** [comment]
@OrangeCrush probably not noticeably locally but it can help :)

File: .dockerignore [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+README.md
+
+*.pyc
+*media/*
+*static/*
+*build/*
+*.DS_STORE
+db.*
+venv*
+node_modules
```
Comment (phalt) [inline .dockerignore]: newline at end of file?

**Comment by phalt** [comment]
👏 amazing stuff @ThisIsAreku just a minor comment then I can merge :)

**Comment by ThisIsAreku** [comment]
@phalt I think it's ready to merge :)

**Pull request merged**